truss/tests/test_validation.py

```diff
@@ -31,3 +33,38 @@ def test_validate_secret_name(secret_name, should_error):
         does_error = True
 
     assert does_error == should_error
+
+
+@pytest.mark.parametrize('cpu_spec, expected_valid', [
+    (None, False),
+    ('', False),
```

Comment:
perhaps add a fractional cpu test here also
```suggestion
    ('', False),
    ('0.5', True),
```

Reply:
great catch, I complete forgot about fractional CPU. Added now.